src/vs/workbench/contrib/chat/browser/chatInputPart.ts

```diff
@@ -595,6 +603,64 @@ export class ChatInputPart extends Disposable implements IHistoryNavigationWidge
 				});
 				widget.ariaLabel = ariaLabel;
 				widget.tabIndex = 0;
+			} else if (attachment.isImage) {
+				let buffer = new Uint8Array(0);
+				if (attachment.value instanceof URI) {
+					const readFile = await this.fileService.readFile(attachment.value);
+					buffer = readFile.value.buffer;
+				} else {
+					buffer = attachment.value as Uint8Array;
+				}
+
+				const blob = new Blob([buffer], { type: 'image/png' });
+				const url = URL.createObjectURL(blob);
+
+				const img = document.createElement('img');
```

Comment:
Using the `dom.$` or `dom.h` helpers is much cleaner. You don't necessarily have to change this but you should look into it for next time, it will save a lot of lines of boilerplate.